src/go/k8s/pkg/resources/pdb.go

```diff
@@ -83,6 +83,10 @@ func (r *PDBResource) obj() (k8sclient.Object, error) {
 			Name:      r.Key().Name,
 			Namespace: r.Key().Namespace,
 		},
+		TypeMeta: metav1.TypeMeta{
```

Comment:
this seems to be unrelated but I had to add it because the change in resource.go meant that some pdb tests started failing.

Reply:
Ah, pod disruption budget!